docs/executor.md

```diff
@@ -407,12 +407,19 @@ Resource requests and other job characteristics can be controlled via the follow
 - {ref}`process-queue`
 - {ref}`process-time`
 
-### Known Limitations
+:::{versionchanged} 24.04.0
+Prior to this version, if `clusterOptions` was specified as a string, multiple options were separated by semicolons to ensure that the job script was formatted correctly:
+```groovy
+clusterOptions = '-t besteffort;--project myproject'
+```
```

Comment:
It would be preferable to keep the current behavior and extend it adding the support for list of options

Reply:
It is still supported, wasn't sure if we should try to phase out this syntax or support both in perpetuity

Reply:
I think we can keep. We have many settings can be both string and list

Reply:
Okay docs updated